src/messages/index.ts

```diff
@@ -21,4 +23,9 @@ export const messages = {
   enabled: 'enabled',
   disabled: 'disabled',
   restart: '\'Restart\' for changes to take effect.',
+  ng2Detected: 'VSCode-Icons has detected an Angular 2 project. ' +
```

Comment:
Do you know something that I don't know? Are going to release Angular 3 in six months?